### PR TITLE
(refactor): Remove Unnecessary Use of VarP()

### LINKED ISSUE #897: `IntVar` should be used instead of `IntVarP` when shorthand is not needed
## Description

For certain commands, `IntVar` and `IntVarP` are used interchangeably (can be applied to other types too). We should use _Var when no shorthand is required.

### PR DESCRIPTION
**Pull request by nickmancari**
Removed unnecessary use of IntVarP(), StringVarP(), BoolVarP() that didn't use or require shorthand parameters and replaced them with IntVar(), StringVar(), and BoolVar().

Closes #897 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [x] Other (please specify)

### Checklist

- [ ] Documentation added for the feature
- [x] CI and all relevant tests are passing
- [x] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: pkg/cmd/cluster/bind/bind.go [hunk 1]
```diff
@@ -60,14 +60,14 @@ func NewBindCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.kubeconfigLocation, "kubeconfig", "", "", opts.localizer.MustLocalize("cluster.common.flag.kubeconfig.description"))
-	cmd.Flags().StringVarP(&opts.appName, "app-name", "", "", opts.localizer.MustLocalize("cluster.bind.flag.appName"))
-	cmd.Flags().StringVarP(&opts.bindingName, "binding-name", "", "", opts.localizer.MustLocalize("cluster.bind.flag.bindName"))
+	cmd.Flags().StringVar(&opts.kubeconfigLocation, "kubeconfig", "", opts.localizer.MustLocalize("cluster.common.flag.kubeconfig.description"))
+	cmd.Flags().StringVar(&opts.appName, "app-name", "", opts.localizer.MustLocalize("cluster.bind.flag.appName"))
+	cmd.Flags().StringVarP(&opts.bindingName, "binding-name", "", opts.localizer.MustLocalize("cluster.bind.flag.bindName"))
```
Comment (wtrocki) [inline pkg/cmd/cluster/bind/bind.go:65]: Should this be changed as well?

**Comment by wtrocki** [comment]
@nickboldt Thank you so much for really valuable contribution. I have reviewed and verified your changes.

**Comment by wtrocki** [comment]
FYI @nickmancari there are still instances in the CLI that use `StringVarP` and `BoolVarP` so I will keep the issue open. 
Do you mind covering other cases as well?

**Comment by nickmancari** [comment]
@wtrocki Thanks! It was my pleasure to contribute!

**Pull request merged**